bot/bot.py

```diff
@@ -1,61 +0,0 @@
-#!/usr/bin/env python
-
-"""
-This module runs the toast every word twitter bot.
-
-Every time it runs, it retrieves a random word from the Wordnik API, 'toasts' it
-into an image of a piece of toast, and tweets that image from @toasteveryword
-(https://twitter.com/toasteveryword)
-"""
-
-import logging
-import config
-from random_word import get_random_word
-from toaster import Toaster
-from twython import Twython
```

Comment:
How do you like twython? I used the `twitter` package for the chanobot and remember pulling my hair out the whole time over poor documentation and lack of features.